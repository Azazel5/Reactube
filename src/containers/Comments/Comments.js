import React, {Component} from 'react'
import CommentsHeader from './CommentsHeader/CommentsHeader'
import AddComment from './AddComment/AddComment'
import Comment from './Comment/Comment'

class Comments extends Component {
    constructor(props) {
        super(props)
        this.state = {}
    }

    render() {
        return (
        <div>
            <CommentsHeader amountComments={this.props.amountComments}/>
            <AddComment />
            <Comment />
            <Comment />
            <Comment />
        </div>
        )
    }
}

export default Comments