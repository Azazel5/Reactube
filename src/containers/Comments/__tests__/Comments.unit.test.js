import React from 'react'
import {shallow} from 'enzyme'
import Comments from '../Comments'

describe('<Comments />', () => {
    test('renders <Comments />', () => {
        const wrapper = shallow(<Comments />)
        expect(wrapper).toMatchSnapshot()
    })

    test('renders <Comments /> with a amountComments prop', () => {
        const wrapper = shallow(<Comments amountComments={123}/>)
        expect(wrapper).toMatchSnapshot()
    })
})