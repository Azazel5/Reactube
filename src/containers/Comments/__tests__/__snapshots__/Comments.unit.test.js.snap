// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Comments /> renders <Comments /> 1`] = `
<div>
  <commentsHeader />
  <addComment />
  <comment />
  <comment />
  <comment />
</div>
`;

exports[`<Comments /> renders <Comments /> with a amountComments prop 1`] = `
<div>
  <commentsHeader
    amountComments={123}
  />
  <addComment />
  <comment />
  <comment />
  <comment />
</div>
`;
